DetailViewController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  DetailViewController.swift
+//  movieViewer
+//
+//  Created by victoria_ku on 6/8/17.
+//  Copyright © 2017 victoria_ku. All rights reserved.
+//
+
+import UIKit
+
+class DetailViewController: UIViewController {
+
+  @IBOutlet weak var postImgView: UIImageView!
+  @IBOutlet weak var titleLabel: UILabel!
+  @IBOutlet weak var overviewLabel: UILabel!
+  @IBOutlet weak var scrollView: UIScrollView!
+  @IBOutlet weak var infoView: UIView!
+
+  var movie: NSDictionary!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+      scrollView.contentSize = CGSize(width: scrollView.frame.size.width, height: infoView.frame.origin.y+infoView.frame.size.height)
+
+
+      let title = movie["title"] as! String
```

Comment:
You are doing some of the parsing here.

Usually, you'll want to create a `Movie` model from your `NSDictionary`, and then parse the dictionary into properties.

that way, you can just do something like `movie.title`